workbench_utils.py

```diff
@@ -292,6 +292,8 @@ def issue_request(
             headers=headers
         )
     if method == 'POST':
+        if '/file/upload' in url:
+            url = url + '?_format=json'
```

Comment:
If you remove this `if`, I can merge this PR.

Reply:
Done (I would have sworn I'd deleted that ...)